style.css

```diff
@@ -124,64 +115,76 @@ ul li:hover #a{
 	width:100%
 }
 
+/* Set up the footer */
 .footer{
```

Comment:
I make some change to the footer, l by adding margin and some update likes fonts, color like what I have decided before